src/index.html

```diff
@@ -30,6 +30,7 @@
     <link rel="icon" sizes="48x48" href="/img/global/icons/favicon.ico" >
     <script src="/javascript/google-tag-manager.js"></script>
     <script id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key=117d80d3-9e4e-4aac-8a1c-1ca9ade4266d"> </script>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css" integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossOrigin="anonymous" />
```

Comment:
Thank you @jarodpeachey. Have you researched whether we need this css file as well as the existing font-awesome link around line 67?:
https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css

It concerns me that we have two font-awesome links. Also, I believe that CSS files should go in the "cssTags" constant around line 67, which get appended to the head after the page has loaded.